jodd-jtx/src/main/java/jodd/jtx/JtxTransactionMode.java

```diff
@@ -249,11 +249,10 @@ public int hashCode() {
 	@Override
 	public String toString() {
 		StringBuilder sb = new StringBuilder();
-		sb.append("jtx{");
-		sb.append(propagationBehavior.toString());
-		sb.append(',').append(readOnlyMode ? "readonly" : "readwrite");
-		sb.append(',').append(isolationLevel.toString());
-		sb.append(',').append(timeout).append('}');
+		sb.append("jtx{").append(propagationBehavior.toString()).append(',')
```

Comment:
Useless `toString()`, that's what StringBuilder.append(Object) does (just that i's protected against NPE and will print "null").